src/CONST.js

```diff
@@ -362,7 +362,7 @@ const CONST = {
             PAYPAL_ME: 'PayPal.me',
             VENMO: 'Venmo',
         },
-        AMOUNT_MAX_LENGTH: 14,
+        AMOUNT_MAX_LENGTH: 11,
```

Comment:
I commented on the deploy blocker, but I think this needs to be 10, our backend shouldn't allow 11 digits (assuming we use the name `amount` for the parameter when calling the API)

Reply:
Ok sorry, small correction. The max is 10 digits when given amount is in cents. So for example you can use enter:

`$12345678.91` because `12345678.91 * 100 = 1234567891` and is 10 digits, but you cannot enter the amount `1234567890`, because `1234567890 * 100 = 123456789000` is 12 digits.

Reply:
Okay I've updated the PR to 10 digits now.